ethier/sign.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/divergencetech/ethier/eth"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+)
+
+var signer *eth.Signer
```

Comment:
What's the global signer used for? This seems very risky for a value that holds and potentially exposes a private key.

Reply:
This was a poor attempt of implementing a singleton pattern for the signer generation. In hindsight, it would have been better to just call `getSigner` in `signAddresses` directly. It has been removed.